### PR TITLE
Added test for GimpPaletteFile

### PR DESCRIPTION
**Pull request by radarhere**

### CODE REVIEW

File: Tests/test_file_gimppalette.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from helper import unittest, PillowTestCase
+
+from PIL.GimpPaletteFile import GimpPaletteFile
+
+
+class TestImage(PillowTestCase):
+
+    def test_sanity(self):
+        fp = open('Tests/images/Test.gpl', 'rb')
```
Comment (hugovk) [inline Tests/test_file_gimppalette.py]: How about a lowercase filename, to be consistent with the others?

File: Tests/test_file_gimppalette.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        GimpPaletteFile(fp)
+        fp.close()
+
+        fp = open('Tests/images/hopper.jpg', 'rb')
```
Comment (hugovk) [inline Tests/test_file_gimppalette.py]: Should it rather use `with open(xxx, 'rb') as fp:` in both these cases?

File: MANIFEST.in [hunk 3]
```diff
@@ -29,6 +29,7 @@ recursive-include Tests *.eps
 recursive-include Tests *.fli
 recursive-include Tests *.ggr
 recursive-include Tests *.gif
+recursive-include Tests *.gpl
```
Comment (hugovk) [inline MANIFEST.in]: The `check-manifest` call seems to be helping :)

File: Tests/test_file_gimppalette.py (hunk 2 above)
Comment (radarhere) [inline Tests/test_file_gimppalette.py]: Okay, sure.

File: MANIFEST.in (hunk 3 above)
Comment (radarhere) [inline MANIFEST.in]: Yep!

File: Tests/test_file_gimppalette.py (hunk 1 above)
Comment (radarhere) [inline Tests/test_file_gimppalette.py]: Fair enough.

**Comment by hugovk** [comment]
Thanks!

**Pull request merged**